tests/h/jinja2_extensions/navbar_data_admin_test.py

```diff
@@ -0,0 +1,38 @@
+from unittest.mock import patch
+
+import pytest
+
+from h.jinja_extensions import navbar_data_admin
+
+STAFF_TABS = ["index", "groups", "mailer", "organizations", "users"]
+
+
+class TestNavbarDataAdmin:
+    @pytest.mark.parametrize("permissive", (True, False))
+    def test_it(self, pyramid_config, pyramid_request, permissive):
```

Comment:
This test doesn't assert the content, just the shape and processing. The contents can change and this would just be a copy of the same data. It wouldn't provide any value.